docs/source/guides/network_node/staking_guide.rst

```diff
@@ -488,6 +490,18 @@ This can help to decrease gas consumption in some operations. To merge two stake
     (nucypher)$ nucypher stake merge --hw-wallet
 
 
+Remove unused sub-stake
+***********************
+
+Combining or editing sub-stakes can lead to 'unused', inactive sub-stakes remaining on-chain. 
```

Comment:
1. What guidance can you offer as to who and when this operation needs to be performed, after every stake editing operation?
2. Usage of the work "combine" here makes me want to rename "merge".

Reply:
1. Remove all inactive (shown using `nucypher stake list --all`) allways if you want pay less as possible for gas,
unlock, increase, merge, slashing could create temporary sub-stakes (which will be inactive soon) or make some sub-stakes inactive, so ideally do some operations and only after all of them - check inactive amd potentially inactive sub-stakes (last period active)
2. Ok